### PR TITLE
Docs/deployment steps

### PR DESCRIPTION
**Pull request by knzeng-e**
This PR provides more details on the README for deployment and initialization of the staking pool.

### CODE REVIEW

File: .env_example [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# #########
+# General deplyoment parameters
```
Comment (jrhender) [inline .env_example:2]:
```suggestion
# General deployment parameters
```

File: .env_example [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# SNAPSHOT_BLOCKNUMBER: The selected block Number to snapshot
+# SNAPSHOT_MIN_BALANCE: Minimum balance for addresses to be included in the snapshot
+# CREDENTIAL_NAME_SPACE: The EW snapshot role 
+#  
+# #############
+CHAINID = 246
+SNAPSHOT_BLOCKNUMBER = 20145052
+SNAPSHOT_MIN_BALANCE = 10000
+EWC_CLAIM_MANAGER_ADDRESS = "0xC3dD7ED75779b33F5Cfb709E0aB02b71fbFA3210"
+CREDENTIAL_NAME_SPACE = "snapshot4.roles.consortiapool.apps.energyweb.auth.ewc"
```
Comment (jrhender) [inline .env_example:26]: I would move these to a separate `.env` so as to not confuse people looking to deploy a staking pool.

File: .env_example [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# #########
+# General deplyoment parameters
+###########
+
+DEPLOYER_PRIV_KEY = <DEPLOYER_PRIVATE_KEY>
+VOLTA_CLAIM_MANAGER_ADDRESS = "0x5339adE9332A604A1c957B9bC1C6eee0Bcf7a031"
+STAKINGPOOL = "0x181A8b2a5AEb25941F6A79b4aE43dBb1968c417A"
```
Comment (jrhender) [inline .env_example:7]: Is this variable used? It doesn't seem to be referenced in `deploy-patron-kyc.js`

File: .env_example [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# #########
+# General deplyoment parameters
+###########
+
+DEPLOYER_PRIV_KEY = <DEPLOYER_PRIVATE_KEY>
+VOLTA_CLAIM_MANAGER_ADDRESS = "0x5339adE9332A604A1c957B9bC1C6eee0Bcf7a031"
```
[the remark this answers is not in the document]
Comment (knzeng-e) [reply .env_example:6]: In real situations, the script is used also to deploy on volta, but we may needs to have a separate dedicated script. So agree, we can remove

File: .env_example (hunk 2 above)
Comment (knzeng-e) [reply .env_example:26]: I totally agree !

File: .env_example (hunk 3 above)
Comment (knzeng-e) [reply .env_example:7]: No it should go with the `.env` file dedicated to snapshot

File: .env_example [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# #########
+# General deployment parameters
+###########
+
+DEPLOYER_PRIV_KEY = <DEPLOYER_PRIVATE_KEY>
+VOLTA_CLAIM_MANAGER_ADDRESS = "0x5339adE9332A604A1c957B9bC1C6eee0Bcf7a031"
+STAKINGPOOL = "0x181A8b2a5AEb25941F6A79b4aE43dBb1968c417A"
+INITIATOR_ADDRESS = <ADMIN_ADDRESS>
+CLAIM_MANAGER_ADDRESS = "0xC3dD7ED75779b33F5Cfb709E0aB02b71fbFA3210"
```
Comment (jrhender) [inline .env_example]: Let's correct this value as well to be the correct EWC value
```suggestion
CLAIM_MANAGER_ADDRESS = "0x23b026631A6f265d17CFee8aa6ced1B244f3920C"
```

File: .env_example (hunk 4 above)
[the remark this answers is not in the document]
Comment (knzeng-e) [reply .env_example]: implemented in commit [3be9e0a](https://github.com/energywebfoundation/staking-pool/pull/61/commits/3be9e0aa2c923b0d0a2caeeee502616eb585843f)

File: .env_example (hunk 3 above)
Comment (knzeng-e) [reply .env_example]: implemented in commit [3be9e0a](https://github.com/energywebfoundation/staking-pool/pull/61/commits/3be9e0aa2c923b0d0a2caeeee502616eb585843f)

File: .env_example (hunk 5 above)
Comment (knzeng-e) [reply .env_example]: implemented in commit [3be9e0a](https://github.com/energywebfoundation/staking-pool/pull/61/commits/3be9e0aa2c923b0d0a2caeeee502616eb585843f)

File: .env_example (hunk 2 above)
Comment (knzeng-e) [reply .env_example]: implemented in commit [3be9e0a](https://github.com/energywebfoundation/staking-pool/pull/61/commits/3be9e0aa2c923b0d0a2caeeee502616eb585843f)

**Review by jrhender — APPROVED** [review]

**Pull request merged**